server/meca/routes.test.js

```diff
@@ -93,7 +93,7 @@ describe('MECA HTTP callback handler', () => {
     await mailer._sendPromise
     expect(mailer.getMails()).toHaveLength(1)
     expect(mailer.getMails()[0]).toMatchObject({
-      text: expect.stringContaining(manuscript.id)
+      text: expect.stringContaining(manuscript.id),
```

Comment:
This looks like a linting change has snuck in. Is this from the pre-commit hook linting or something running locally?

Reply:
Yes it was added when I committed, looking to how to get rid of that setting.